### PR TITLE
drop old celery/djcelery tables we no longer need

### PR DESCRIPTION
**Pull request by ryanpetrello**

### CODE REVIEW

File: awx/main/migrations/0050_v340_drop_celery_tables.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+    dependencies = [
+        ('main', '0049_v330_validate_instance_capacity_adjustment'),
+    ]
+
+    operations = [
+        migrations.RunSQL([
+            ("DROP TABLE IF EXISTS {} CASCADE;".format(table))
+        ])
+        for table in ('celery_taskmeta', 'celery_tasksetmeta', 'djcelery_crontabschedule',
```
Comment (ryanpetrello) [inline awx/main/migrations/0050_v340_drop_celery_tables.py:17]: @matburt any concerns over the performance/speed of dropping any of these tables on upgrade?  _maybe_ `celery_taskmeta`?  My understanding is that celery periodically cleans these, so I don't expect _too much_ by way of table size.

**Review by wwitzel3 — APPROVED** [review]
This LGTM, I'd feel really great about it if we verified scheduled jobs do what they are supposed to do. I know internally beat uses a shelve , but I'd like to make sure before we merge it.

File: awx/main/migrations/0050_v340_drop_celery_tables.py (hunk 1 above)
Comment (matburt) [reply awx/main/migrations/0050_v340_drop_celery_tables.py:17]: no concerns from me here.

**Review by matburt — APPROVED** [review]

**Comment by ryanpetrello** [comment]
@wwitzel3 I ran integration and also confirmed that schedules work fine w/o these tables.

**Comment by ryanpetrello** [comment]
regate

**Comment by ryanpetrello** [comment]
regate

**Comment by ryanpetrello** [comment]
regate

**Pull request merged**